### DISCUSSION TITLE
Running Storybook v6.x in Node18

### DISCUSSION THREAD

**Discussion by sethdumaguin** [storybookjs/storybook#24461] [Help]
Hi there! My app is running in node18 with webpack 4's latest minor version ([4.47](https://github.com/webpack/webpack/releases/tag/v4.47.0)) and it's currently working. 

However, when I am running `yarn storybook` for my app, I am getting the `Error: error:0308010C:digital envelope routines::unsupported`. I know version webpack4.47 was just released a month ago, but will storybook's v6 be supporting this new webpack version? 

Or is there a way I can run storybook v6 in node18 and webpack4? Appreciate the help!

**Comment by valentinpalkovic** [comment]
We are not actively maintaining Storybook v6.x anymore. You could use the [overrides](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#overrides) field in your package.json to install a dedicated version of webpack, independently of what Storybook defines internally.
